agents/community/langgraph-agents/beeai-provider-unmanaged.yaml

```diff
@@ -0,0 +1,5 @@
+manifestVersion: 1
```

Comment:
Let's not add "unmanaged" provider to each agent, they are all the same - they target the port, not the agent. There can be maybe one unmanaged provider in the "agents" folder if necessary.

Reply:
When you wanna test more agents locally, you need to have move yaml files and run them on the different ports.